src/core/automount.c

```diff
@@ -965,13 +965,16 @@ static const char *automount_sub_state_to_string(Unit *u) {
         return automount_state_to_string(AUTOMOUNT(u)->state);
 }
 
-static bool automount_check_gc(Unit *u) {
+static bool automount_can_gc(Unit *u) {
```

Comment:
logind and machined have similarly named functions, i figure we should rename them all in one go then, too?

Also, shouldn't this be "automount_may_gc()" rather? sounds more polite

Reply:
OK. I renamed to "may_gc" everywhere, and made similar changes to machined and logind.